### PR TITLE
VV improvements: rotate, varedit updates, link to VV'd object

### PR DESCRIPTION
**Pull request by DominikPanic**
Rohesie's request

### CODE REVIEW

File: code/modules/admin/verbs/datumvars.dm [hunk 1]
```diff
@@ -927,6 +929,26 @@
...
+			return
+
+		var/atom/A = locate(href_list["rotatedatum"])
+		if(!istype(A))
+			to_chat(usr, "This can only be done to instances of type /atom")
+			return
+
+		switch(href_list["rotatedir"])
+			if("right")
+				A.setDir(turn(A.dir, -45))
```
Comment (spookydonut) [inline code/modules/admin/verbs/datumvars.dm]: some things won't play nicely with 45 degree rotations

Comment (DominikPanic) [reply code/modules/admin/verbs/datumvars.dm]: This is /tg/ code, what would you suggest?

File: code/modules/admin/verbs/datumvars.dm [hunk 2]
```diff
@@ -693,6 +693,8 @@
 			return
 
 		M.fully_replace_character_name(M.real_name, new_name)
+		vv_update_display(M, "name", new_name)
```
Comment (spookydonut) [inline code/modules/admin/verbs/datumvars.dm]: shouldnt this be in #1384

Comment (DominikPanic) [reply code/modules/admin/verbs/datumvars.dm]: This is VV/admin tools related, that one is for a game proc

**Pull request merged**